app/src/test/java/com/codesoom/assignment/utils/JwtUtilTest.java

```diff
@@ -0,0 +1,86 @@
+package com.codesoom.assignment.utils;
+
+import com.codesoom.assignment.errors.InvalidTokenException;
+import io.jsonwebtoken.Claims;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("JwtUtil")
+class JwtUtilTest {
+    private final static String SECRETE_KEY = "12345678901234567890123456789010";
+    private final static String VALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9" +
+            ".cjDHNEbvUC6G7AORn068kENYHYnOTIaMsgjD0Yyygn4";
+    private final static String INVALID_TOKEN = VALID_TOKEN+"WRONG";
```

Comment:
```suggestion
    private final static String INVALID_TOKEN = VALID_TOKEN + "WRONG";
```

취향이지만 중간에 빈 칸을 추가하면 더 잘보일 것 같습니다. 자동정렬할 경우 이렇게 안띄워주나요?

Reply:
제가 마지막에 수정하다보니 누락했네요 수정했습니다.